### PR TITLE
chore: minor fixes

### PR DESCRIPTION
**Pull request by octonato**
Notices these two minor issues when preparing the brown bag presentation

### CODE REVIEW

File: sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Action with the same name in a different package.
+ */
+public class EchoAction extends Action {
+
+  @GetMapping("/echo2/message/{msg}")
+  public Effect<Message> stringMessage(@PathVariable String msg) {
+    return effects().reply(new Message(msg));
+  }
+}
```
Comment (efgpinto) [inline sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java:33]: Isn't this file already in [main](https://github.com/lightbend/kalix-jvm-sdk/blob/dc06f25309d75c55ea1faa76c497def6fc085320/sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java#L27) ? Or what is the difference here?

File: sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/ProtoDescriptorGenerator.scala [hunk 2]
```diff
@@ -67,4 +67,8 @@ private[impl] object ProtoDescriptorGenerator {
     }
     fd
   }
+
+  def fileDescriptorName(packageName: String, name: String) = {
+    packageName.replace(".", "/") + "/" + name + ".proto"
+  }
```
Comment (efgpinto) [inline sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/ProtoDescriptorGenerator.scala:73]: also this?

**Review by efgpinto — COMMENTED** [review]
Seems like the PR has some unrelated changes that are already in main? Or I'm I missing something?

File: sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java (hunk 1 above)
Comment (octonato) [reply sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java:33]: I think I need to rebase this branch.

Comment (octonato) [reply sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java]: I did a rebase. Both the EchoAction nor the ProtoDescriptorGenerator weren't something that I added myself. They are not removed for the diff.

**Review by efgpinto — APPROVED** [review]

**Pull request merged**